design/SAS/R_weighted.R

```diff
@@ -0,0 +1,33 @@
+# Weighted MMRM and LS means-
```

Comment:
looking at the folder name here, this can be confusing - maybe move non-SAS files to another folder? e.g. `comparisons` or `R` ?